translate/cloud-client/hybrid_glossaries/README.rst

```diff
@@ -0,0 +1,97 @@
+.. This file is automatically generated. Do not edit this file directly.
```

Comment:
I wonder if these should all be under `translate/hybrid_glossaries` rather than `translate/cloud-client/hybrid_glossaries`. 

@beccasaurus